nginx-controller/nginx/nevermerger.go

```diff
@@ -0,0 +1,43 @@
+package nginx
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+)
+
+type NeverMerger struct {
```

Comment:
> Adding a additional collision check afterwards will only be redundant self validation, which should be put into the unittests of the Merger implementation.

I think  a conditional statement in the configurator is what we need:
**if** merging is enabled then do the merging, using one of the implementations
**else** do the hostname checking, if any error -- ignore the Ingress and report the error

> Edit: We could also rename this component to something like Deduplicator, if this matches the behavior better ;)

but it will still have the interface of the meger